api/src/main/java/com/codeforcommunity/dto/team/TeamGoalDataResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.codeforcommunity.dto.team;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class TeamGoalDataResponse extends TeamDataResponse {
+  private final List<GoalResponse> goals;
+
+  public TeamGoalDataResponse(
+          Integer id,
+          String teamName,
+          String bio,
+          Boolean finished,
+          List<GoalResponse> goals,
+          Timestamp createdAt,
+          Timestamp deletedAt) {
+    super(id, teamName, bio, finished, createdAt, deletedAt);
+    this.goals = goals;
+  }
```

Comment:
Also no empty constructor

Reply:
Actually, the empty constructor is only necessary for DTOs that are coming from requests, because this DTO is just being used for responses we're only going to be constructing it in the normal way so we don't need the empty constructor.